tools/topology/topology2/include/components/aec.conf

```diff
@@ -0,0 +1,160 @@
+#
+# Common widget aec
+#
+# A generic aec widget. All attributes defined herein are namespaced
+# by alsatplg to "Object.Widget.aec.N.attribute_name"
+#
+# Usage: this component can be used by instantiating it in the parent object. i.e.
+#
+# 	Object.Widget.aec."N" {
+# 		index		1
+# 		type		"effect"
+#	}
+#
+# Where N is the unique instance number for the aec object within the same alsaconf node.
+Define {
+	AEC_UUID	"a6:a0:80:b7:9f:26:6f:46:b4:77:23:df:a0:5a:f7:58"
+}
+
+Class.Widget."aec" {
+	#
+	# Pipeline ID for the aec widget object
```

Comment:
the AEC uses two sources (reference + capture), and has one sink pin. What would the pipeline look like here. Is this a pipeline with a single module?

Reply:
Defined two pins in this file. And the binding info is defined in pipeline